0.initial-setup.sh

```diff
@@ -6,7 +6,7 @@ RES=scripts
 
 rm -rf output
 mkdir output
-chown -R ec2-user:ec2-user output
+chown -R $SUDO_USER. output
```

Comment:
Not sure if $SUDO_USER is available if you are logged in as `root`. Wouldn't $USER be sufficient here?

Reply:
ah because it's run with `sudo` in the instructions. We could do `USR=${SUDO_USER:-$USER}` and that would cover both cases.